django_stubs_ext/django_stubs_ext/aliases.py

```diff
@@ -2,9 +2,15 @@
 
 if typing.TYPE_CHECKING:
     from django.db.models.query import _T, _QuerySet, _Row
+    from django.utils.functional import _StrOrPromise, _StrPromise
```

Comment:
It's fair to have this re-exported in aliases (I think it should work?)

```suggestion
    from django.utils.functional import _StrOrPromise as StrOrPromise, _StrPromise as StrPromise
```

Reply:
This fails the mypy check https://github.com/typeddjango/django-stubs/actions/runs/3076096523/jobs/4970014010. We will need to allow implicit_reexport for `django_stubs_ext.aliases` if we do it this way.

Reply:
Actually, never mind. Adding these aliases to `__all__` should fix the CI.

Reply:
@flaeppe Mypy is very specific about [what counts as a re-export](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport):

```python3
# This won't re-export the value
from foo import bar

# Neither will this
from foo import bar as bang

# This will re-export it as bar and allow other modules to import it
from foo import bar as bar

# This will also re-export bar
from foo import bar
__all__ = ['bar']
```

Reply:
If I understand correctly, mypy does not mention that this below is discouraged, right? Though we may as well define a type alias at this point.

```python
from foo import bar as bang
__all__ = ['bang']
```